app/crud/category.py

```diff
@@ -0,0 +1,60 @@
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy import select, delete
+from uuid import UUID
+from typing import List, Optional
+
+from app.models.category import Category
+from app.schemas.category import CategoryCreate, CategoryUpdate
+
+
+async def create_category(db: AsyncSession, category_data: CategoryCreate) -> Category:
```

Comment:
crud에 전체 다 `CategoryResponse`로 해야하지 않을까요?

@trivialcoding6  제가 이쪽 지식은 없어서 계속 태그하겠습니다 ㅠㅠ

```suggestion
async def create_category(db: AsyncSession, category_data: CategoryCreate) -> CategoryResponse:
```

Reply:
@GideokKim 
넵 이 부분도 기덕님이 남겨주신게 맞습니당

Reply:
관련해서 전부 수정 했습니다